pwiz_tools/Shared/zedgraph/ZedGraph/GraphPane.cs

```diff
@@ -21,13 +21,20 @@
 using System.Drawing;
 using System.Drawing.Drawing2D;
 using System.Collections;
+using System.Collections.Generic;
 using System.Drawing.Imaging;
 using System.Windows.Forms;
 using System.Diagnostics;
 using System.IO;
 using System.Runtime.Serialization;
 using System.Security.Permissions;
 using System.ComponentModel;
+using System.Diagnostics.Eventing.Reader;
```

Comment:
This is a lot of code to add to an already large and central class. Try to separate out most of this code into another file or more.